base_location/views/res_country_view.xml

```diff
@@ -20,7 +19,7 @@
                     class="oe_stat_button"
                     icon="fa-globe"
                     type="action"
-                    context="{'default_country_id': active_id, 'search_default_country_id': active_id}"
+                    context="{'default_country_id': id, 'search_default_country_id': id}"
```

Comment:
This is not working properly: all ZIPs are visible from all countries.

Reply:
I found the reason, it is not caused by the migration, it is not working on V16 runboat either. It was because country_id was not in the search, I added it.